target/scripts/wrapper/postsrsd.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+function postsrsd_stop
+{
+  /etc/init.d/postsrsd stop # does not properly stop postsrsd process
+  # shellcheck disable=SC2046
+  kill $(pidof postsrsd)
+}
+
+trap postsrsd_stop EXIT
```

Comment:
I remember that other wrappers had traps for SIGINT and SIGTERM as well - I think Supervisord will send these signals actually, so shouldn't we have these trapped as well?

Reply:
👍🏼